libbpf-cargo/Cargo.toml

```diff
@@ -38,6 +38,7 @@ structopt = "0.3"
 semver = "0.9"
 tempfile = "3.1"
 thiserror = "1.0"
+memmap2 = "0.3.0"
```

Comment:
Shouldn't the old `memmap = 0.7.0` dep be removed too?

Reply:
> Shouldn't the old `memmap = 0.7.0` dep be removed too?

Oops, I think this got added back when I was rebasing. I'll remove it again!